src/adapters/api-blueprint-adapter.coffee

```diff
@@ -298,9 +297,9 @@ legacyResourcesFrom1AResource = (legacyUrlConverterFn, resource, sourcemap, opti
       legacyResource.model = {}
 
     legacyResource.resourceParameters = resourceParameters or []
-    legacyResource.actionParameters   = actionParameters or []
+    legacyResource.actionParameters   = getParametersOf(action, options) or []
 
-    legacyResource.parameters = actionParameters or resourceParameters or []
+    legacyResource.parameters = legacyResource.resourceParameters.concat(legacyResource.actionParameters)
```

Comment:
I can also override the URI Template in an action so it may even contain parameters that do not belong in resource. There can also be parameters described in both a resource and an action and this code will duplicate them.